### PR TITLE
[JN-1258] Clean up admin navbar on mobile

### PR DESCRIPTION
**Pull request by MatthewBemis**
#### DESCRIPTION (include screenshots, and mobile screenshots for participant UX)

To improve the kit scanning experience, this cleans up the navbar on the admin tool when viewing on a mobile device.

Also adds a local storage entry to remember when the sidebar was hidden, which is particularly important on mobile because the app is essentially unusable unless the sidebar is collapsed.

Mobile before (navbar hidden/expanded):
Screenshot 2024-09-28 at 3 04 20 PMScreenshot 2024-09-28 at 3 04 33 PM

Mobile after (navbar hidden/expanded):
Screenshot 2024-09-28 at 4 03 09 PMScreenshot 2024-09-28 at 4 03 20 PM

Desktop after:
Screenshot 2024-09-28 at 4 04 56 PM

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

Confirm the new navbar and sidebar functionality still works and there aren't any regressions
Confirm that the new styling looks good on mobile

### CODE REVIEW

File: ui-admin/src/study/kits/kitcollection/KitScanner.tsx [hunk 1]
```diff
@@ -287,14 +287,14 @@ export const KitScanner = ({ studyEnvContext }: { studyEnvContext: StudyEnvConte
...
             }}/>
-          { showKitScanner &&
-          <BarcodeScanner
-            expectedFormats={['code_128']}
-            onError={error => setReturnTrackingNumberError(error)}
-            onSuccess={result => {
-              setReturnTrackingNumber(result.rawValue)
-              setShowReturnTrackingNumberScanner(false)
-            }}/>
+          { showReturnTrackingNumberScanner &&
```
Comment (MatthewBemis) [inline ui-admin/src/study/kits/kitcollection/KitScanner.tsx:290]: driveby bug fix

**Review by connorlbark — APPROVED** [review]
nice! mobile is a lot less frustrating now

**Review by devonbush — APPROVED** [review]
nice
